scripts/remote_map_planets_check.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash -e
+# remote_map_planets_check.sh - Make global planet test maps for both 05m and 06m resolutions
+#
+# usage: remote_map_planets_check
+#
+# Loops over all recipes and makes global maps of them for the two resolutions
+# This checks that the tiled and single grid versions are similar.  We use the
+# candidate server which must pass all tests before overwriting oceania.
+#
+list=(mars   mercury moon   pluto  venus)
+type=(relief relief  relief relief relief)
```

Comment:
Why is it neccesary this line? Can't just set it to "relief" in the lines 18 and 20?

Reply:
Might get lunar gravity at some point so thought we should plan for that

Reply:
Ok, and maybe some images would good.